### DISCUSSION TITLE
PR Merge

### DISCUSSION THREAD

**Discussion by Molza01** [vishanurag/Canvas-Editor#920] [General]
Hey @vishanurag , I’ve submitted a PR that addresses the Alignment and Changes of the Submit Button. Can you please review it when you get a chance? Let me know if there’s anything that needs adjustment before merging. Thanks!

**Comment by DevROHIT11** [comment]
Heyy @vishanurag I just raised a PR #796 I completed the changes which i mean  to do for. Can u plz check my PR request and would accept it . If their is need to do some changes please tell me I will do that .
Thank you !!
